sdk/appcenter/src/test/java/com/microsoft/appcenter/utils/context/AuthTokenContextTest.java

```diff
@@ -118,8 +121,8 @@ public void setAuthTokenTest() {
         mAuthTokenContext.setAuthToken(null, null, null);
 
         /* Verify that listener is called on empty token. */
-        verify(mockListener, times(1)).onNewAuthToken(isNull(String.class));
-        verify(mockListener, times(1)).onNewUser(isNull(String.class));
+         verify(mockListener).onNewAuthToken(isNull(String.class));
+        verify(mockListener).onNewUser(isNull(UserInformation.class));
```

Comment:
Comment was not addressed: we need better than a null check, we need the test to pass a long homeAccountId value and verify the UserInformation contains the short accountId.

Reply:
Updated